examples/php/composer.json

```diff
@@ -7,7 +7,7 @@
   },
   "autoload": {
     "psr-4": {
-      "": "route_guide/"
+      "": "."
```

Comment:
If you remove it, you'll also need to add code below into the router_guide/route_guide_client.php to make the example work.
```
@include_once dirname(__FILE__).'/GPBMetadata/RouteGuide.php';
@include_once dirname(__FILE__).'/Routeguide/RouteGuideClient.php';
@include_once dirname(__FILE__).'/Routeguide/RouteNote.php';
@include_once dirname(__FILE__).'/Routeguide/RouteSummary.php';
@include_once dirname(__FILE__).'/Routeguide/Rectangle.php';
@include_once dirname(__FILE__).'/Routeguide/Point.php';
@include_once dirname(__FILE__).'/Routeguide/Feature.php';
```